optionals/compat_optre/cfgVehicles.hpp

```diff
@@ -0,0 +1,503 @@
+class CfgVehicles {
+	class Man;
+    class CAManBase: Man
+    {
+        class ACE_SelfActions
+        {
```

Comment:
Why is this even added to optionals? Why it can't be added into said add-on?
24.02.2016 2:38 PM "jonpas" <notifications@github.com> napisał(a):

> In optionals/compat_optre/cfgVehicles.hpp
> <https://github.com/acemod/ACE3/pull/2859#discussion_r53938512>:
>
> > @@ -0,0 +1,503 @@
> > +class CfgVehicles {
> > +	class Man;
> > +    class CAManBase: Man
> > +    {
> > +        class ACE_SelfActions
> > +        {
>
> Opening brace on the same line as class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acemod/ACE3/pull/2859/files#r53938512>.
>